packages/cli/src/prompts/method-params.ts

```diff
@@ -59,18 +59,20 @@ function getCommandProps(
     return {
       ...accum,
       [arg.name]: {
-        message: arg.name ? `${arg.name} (${arg.type}):` : `(${arg.type}):`,
+        message: argLabel(arg),
```

Comment:
Note that the previous string had a colon at the end, and `argLabel` doesn't (which is okay because in the other place where it's used there was no colon).
```suggestion
        message: argLabel(arg) + ':',
```

Reply:
Good catch!